### PR TITLE
Autodesk.DWGTrueView version 24.1.154.0

### PR DESCRIPTION
**Pull request by TechnicaVivunt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/39305)

### CODE REVIEW

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
+PackageVersion: 4.0.19.0
+PackageLocale: en-US
+Publisher: Autodesk, Inc.
+PackageName: AutoDesk DWG TrueView
+License: Copyright 2015 Autodesk, Inc. All rights reserved.
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
Copyright: Copyright 2015 Autodesk, Inc. All rights reserved.
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml:5]:
```suggestion
PackageVersion: 24.1.154.0
```
image

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 24.1.154.0
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.yaml:5]:
```suggestion
PackageVersion: 24.1.154.0
```

File: manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Autodesk,Inc.AutoDeskDWGTrueView
+PackageVersion: 4.0.19.0
+InstallerSwitches:
+  Silent: --silent
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://efulfillment.autodesk.com/NetSWDLD/2022/ACD/C1E477AF-DE1B-34E4-B76B-505C7B8FB980/SFX/DWGTrueView_2022_English_64bit_dlm.sfx.exe
+  InstallerSha256: E2E7E35B27FBECA7587E74969DEFE75C155FA309E10C97BA66AAD45F873975B6
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutoDeskDWGTrueView/4.0.19.0/Autodesk,Inc.AutoDeskDWGTrueView.installer.yaml:12]:
```suggestion
  InstallerSha256: E2E7E35B27FBECA7587E74969DEFE75C155FA309E10C97BA66AAD45F873975B6
  ProductCode: '{C1E477AF-DE1B-34E4-B76B-505C7B8FB980}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
You will need to update the folder the manifest is in because of the ID change

**Comment by TechnicaVivunt** [comment]
Installer flag will not work until the initial exe extracts working files it seems. Winget would likely need to accommodate for this somehow.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This doesn't seem to install silently because of this prompt:
image

You will most likely need to contact the publisher and see if there are installer switches to automatically set the installation directory.

**Comment by DW-42** [comment]
TTBOMK: DWG TrueView 2022 does not have the option to perform an unattended install, straight out of the box. Your need to  extract the contents of the DWGTrueView_2022_English_64bit_dlm.sfx.exe file, and only then can you run setup.exe. Unfortunately winget is expecting _direct_ access to the installer and thus cannot handle this senerio.

see: [https://silentinstallhq.com/autodesk-dwg-trueview-2022-silent-install-how-to-guide/](https://silentinstallhq.com/autodesk-dwg-trueview-2022-silent-install-how-to-guide/)

**Pull request closed without merging**